Training/TPU-v5p/Llama2-7B-Maxtext/TPU_v4_user_guide.md

```diff
@@ -1,5 +1,35 @@
 # Instructions for training Llama2-7B-Maxtext on TPU v4
```

Comment:
Should this be under `Training/TPU-v4/Llama2-7B-Maxtext/README.md`?

Reply:
Done.